lib/player.js

```diff
@@ -385,6 +385,26 @@ goog.requireType('shaka.routing.Payload');
  */
 
 
+/**
+ * @event shaka.Player.StallDetectedEvent
+ * @description Fired when a stall in playback is detected by the StallDetector.
+ *     Not all stalls can be caused by gaps in the buffered ranges.
```

Comment:
"can be caused"? I assume you mean "are caused"

Reply:
Fixed, thanks!